tests/test_utils.py

```diff
@@ -47,12 +45,12 @@ def test_read_labels():
     assert(isinstance(labels, dict))
 
 
-def test_write_labels():
+@pytest.fixture(scope='session')
+def test_write_labels(tmpdir):
     label = {1: 1, 2: 2}
-    d = TemporaryDirectory()
-    file_path = d.name + "/test.csv"
-    utils.write_labels(label, file_name=file_path)
-    assert(os.path.exists(file_path))
+    fn = tmpdir.mkdir("data").join("test.csv")
```

Comment:
No need to create a new directory, use tmpdir directly.
Also, don't change the name of the variable